apisix/plugins/error-log-logger.lua

```diff
@@ -237,10 +237,15 @@ local function send_to_skywalking(log_message)
     httpc:set_timeout(config.timeout * 1000)
 
     local entries = {}
+    local service_instance_name = config.skywalking.service_instance_name
+    if service_instance_name == "$hostname" then
+        service_instance_name = core.utils.gethostname()
+    end
```

Comment:
Hello there, we need a test case to ensure it work as expected. :)

Reply:
@shreemaan-abhishek

Reply:
@leslie-tsang, isn't this sufficient 🤔

https://github.com/dickens7/apisix/blob/da3a8db37b84bc45c8edd98ac80be2c5a64a9d99/t/plugin/skywalking-logger.t#L239-L247

Reply:
We need a test case for `error-log-logger`

Reply:
added